### ISSUE TITLE
Any plans to migrate Parcel to V2?

### ISSUE THREAD

**Issue by aboveyunhai** [weiran-zsd/dts-cli#66]
### Current Behavior

![image](https://user-images.githubusercontent.com/35160613/136632287-5dfca5af-d28f-47a4-9ebf-4607c247cce8.png)

The weird part is that inside the `devDependencies`, `parcel` is `^1.12.3`, parcel is installed as `1.12.4`.

The current project will always fail in `/example` project due to the `Parcel` bug. 
It's been discussed for a while but it seems like this forks does not address this critical issue.
Have to do hard roll back to `1,12.3` by `npm i parcel@1.12.3`, which creates some questionable warning

![image](https://user-images.githubusercontent.com/35160613/136632399-5c9b5901-969c-4bd8-97ce-ac40264ad76c.png)

### Expected behavior
Run `npm run start` without problems

### Your environment
windows

```text

```

**Comment by aladdin-add** [comment]
yes, but i slightly prefer vite.js;  anyway, parcel v1 should have been went away.
// cc @sidwebworks

**Comment by aladdin-add** [comment]
should have been fixed in v0.19.1. :🎉

**Comment by aboveyunhai** [comment]
> should have been fixed in v0.19.1. :🎉

Apologize for the naïve question, is it possible to update existing package via cli?
My current package is still using `parcel`, I did not find any useful doc to update without recreating a new project.

**Comment by aboveyunhai** [comment]
> No straightforward way to get it. you can just create a new one, and copy the `example/` to your existing package. 😄

No worry, I did understand the problem behinds it,  just want to get the confirmation so I don't miss anything besides doc. Thx for your work!
